tools/create_snapshot_artifact/main.py

```diff
@@ -11,12 +11,12 @@
 import tempfile
 from pathlib import Path
 
-from host_tools.cargo_build import get_firecracker_binaries
-
 # Hack to be able to import testing framework functions.
 sys.path.append(os.path.join(os.getcwd(), "tests"))  # noqa: E402
 
 # pylint: disable=wrong-import-position
+from host_tools.cargo_build import get_firecracker_binaries
+
 # The test infra assumes it is running from the `tests` directory.
 os.chdir("tests")
```

Comment:
Actually, it seems there's already a workaround for this (didnt know that when I wrote my ab_test.py script). Can we use the same hack everywhere? E.g. either the "os.chdir" to get it to resolve the imports, or the `path.append`?

Reply:
There's another way, we could add `env PYTHONPATH=tests` here: https://github.com/firecracker-microvm/firecracker/blob/main/.buildkite/pipeline_cross.py#L42

Or we could even add it always `devtool::cmd_sh`

Reply:
The `env PYTHONPATH=tests` thing was the first solution I tried but then I saw that there was an existing fix with the sys.path.append and with this we can run the scripts standalone inside the container as well.

Reply:
I am more partial to the environment variable, but this usually has worked well in the past, so no reason to change it.